<launch>
   <arg name="drone_id" default="drone1"/>
   <arg name="sp_out_hz" default="100"/>
   <arg name="checkup_hz" default="5"/>
   <arg name="sp_gcs_hz_min" default="2"/>
   <arg name="checkup_hz_min" default="5"/>

   <group ns="$(arg drone_id)">
   	<node pkg="bridge_px4" type="setpoint_publisher" name ="setpoint_publisher" output="screen">
	   </node>

      <include file="$(find mavros)/launch/node.launch">
         <arg name="pluginlists_yaml" value="$(find mavros)/launch/px4_pluginlists.yaml" />
         <arg name="config_yaml" value="$(find mavros)/launch/px4_config.yaml" />

         <arg name="fcu_url" value="/dev/ttyS4:921600" />
         <arg name="gcs_url" value="" />
         <arg name="tgt_system" value="1" />
         <arg name="tgt_component" value="1" />
         <arg name="log_output" value="screen" />
         <arg name="fcu_protocol" value="v2.0" />
         <arg name="respawn_mavros" default="false" />
      </include>
   </group>
</launch>
